packages/pigeon/platform_tests/ios_swift_unit_tests/ios/RunnerTests/AllDatatypesTests.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+@testable import Runner
+
+class AllDatatypesTests: XCTestCase {
```

Comment:
This code needs to follow [the Google Swift style guide](https://google.github.io/swift/); notably, all the indentation here is using 4 spaces instead of two. (We don't have an autoformatter for Swift in our tooling because it's still non-trivial to set up swift-format).

Reply:
Fixed